docs/qudits.ipynb

```diff
@@ -102,34 +100,45 @@
    "metadata": {
     "id": "6b3c6308ddd3"
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "0 (d=3): ───[+1]───\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "import cirq\n",
+    "import numpy as np\n",
     "\n",
-    "class QutritPlusGate(cirq.SingleQubitGate):\n",
+    "class QutritPlusGate(cirq.Gate):\n",
+    "    \"\"\"A gate that adds one in the computational basis of a qutrit.\n",
+    "    \n",
+    "    This gate acts on three-level systems. In the computational basis of\n",
+    "    this system it enacts the transformation U|x> = |x + 1 mod 3>, or\n",
+    "    in other words U|0> = |1>, U|1> = |2>, and U|2> = |0>.\n",
+    "    \"\"\"\n",
+    "    \n",
     "    def _qid_shape_(self):\n",
+    "        # By implementing this method this gate implements the\n",
```

Comment:
Optional:  Can this be a docstring with triple quotes rather than line comments?  Same below.